charts/dev/chevereto/values.yaml

```diff
@@ -10,6 +10,7 @@ securityContext:
 PodSecurityPolicy:
   runAsUser: 0
   runAsGroup: 0
+  fsGroup: 0
```

Comment:
nope don't use fsGroup 0 please. It's never needed.

We just need to add the required persistence everywhere and if that's not enough add "RootOnlyFileSystem = false"